specification/schemaregistry/data-plane/Microsoft.EventHub/stable/2022-10/schemaregistry.json

```diff
@@ -523,14 +508,18 @@
     },
     "SchemaVersions": {
       "type": "object",
-      "description": "Array received from the registry containing the list of versions for specific schema.",
+      "description": "Object received from the registry containing the list of schema versions and link to next batch page.",
       "properties": {
-        "schemaVersions": {
+        "Value": {
           "type": "array",
-          "description": "Array of schema groups.",
+          "description": "Array of schema version integers.",
           "items": {
             "$ref": "#/definitions/SchemaVersion"
           }
+        },
+        "NextLink": {
```

Comment:
This must be `nextLink`. JSON properties are case-sensitive and should be camelCase.

Reply:
You're also missing the x-ms-pageable. See https://github.com/Azure/autorest/tree/main/docs/extensions#x-ms-pageable. See https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#collections for our guidelines on pageables.

Reply:
Will this cause breaking changes? I started aligning the types to match the x-ms-pageable field names but that caused the CI to require a breaking change review

Reply:
It's only a breaking change to SDKs. Have you shipped a track 2 SDK on the 2021-10 API? If not, I'd make it now because once you ship a track 2 SDK, making collections pageable later is breaking.

The whole point of the nextLink was to make it pageable, but the x-ms-pageable is also required or the SDKs won't generate the methods correctly. A fetch would only get the first page, and no method would even be generated to get a subsequent page if customers used the nextLink themselves. In short, there's no way it'd be pageable, which is normally automatic like with C# SDKs:

```c#
await foreach (var result in client.GetFoos())
{
}
```